ERCS/erc-0000.md

```diff
@@ -0,0 +1,176 @@
+
+The interface is intentionally neutral and does not impose conditions on:
+
+- **Access Control**: Token issuers determine who is authorized to call `crosschainMint()` and `crosschainBurn()`.
+- **Zero Amount Calls**: Token issuers decide whether to allow or revert calls with zero amounts.
+
+## Backwards Compatibility
+
+This proposal is fully backwards compatible with [ERC-20](./eip-20.md).
+
+As discussed in the Motivation section, a minimal, flexible cross-chain standard interface is necessary. The problem becomes larger as more tokens are deployed without a standardized format.
+
+- Upgradable tokens can be upgraded to implement the new interface.
+- Non-upgradable tokens cannot implement the interface on the token itself. They can still migrate to a standard-compliant version using a lockbox mechanism, as proposed by xERC-20. The idea is to lock non-mintable tokens and mint the same amount of interface-compliant tokens. The bridge contract can act as a lockbox on the native chain.
+
+Bridge contracts will also need an upgrade to integrate with the interface. Most popular bridges in terms of liquidity at the date are upgradable or use tokens that can be upgraded to point to new bridge contracts. 
+
+1. Arbitrum One ($5.23B)
+2. Portal Wormhole ($2.43B)
+3. Polygon PoS ($2.15B)
+4. OP Mainnet ($2.09B)
+5. Mantle ($983M)
+6. LayerZero OFTs ($465M)
+7. Omnibridge ($449M)
+8. Base ($350M)
+9. Scroll ($128M)
+10. zkSync ($77M)
+
+Source: L2Beat (10/31/24)
+
+## Reference Implementation
+
+```solidity
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.25;
+
+abstract contract CrosschainERC20 is ERC20, IERCxxx {
+		/// @notice Address of the TOKEN_BRIDGE contract that is allowed to mint/burn tokens.
+    address public immutable TOKEN_BRIDGE;
+
+    /// @notice Constructor to set the TOKEN_BRIDGE address.
+    /// @param _tokenBridge Address of the TOKEN_BRIDGE.
+    constructor(address _tokenBridge) {
+        require(_tokenBridge != address(0), "Invalid TOKEN_BRIDGE address");
+        TOKEN_BRIDGE = _tokenBridge;
+    }
+    
+    /// @notice A modifier that only allows the TOKEN_BRIDGE to call
+    modifier onlyTokenBridge() {
+        if (msg.sender != TOKEN_BRIDGE) revert Unauthorized();
+        _;
+    }
+
+    /// @notice Allows the TOKEN_BRIDGE to mint tokens.
+    /// @param _to     Address to mint tokens to.
+    /// @param _amount Amount of tokens to mint.
+    function crosschainMint(address _to, uint256 _amount) external onlyTokenBridge {
+        _mint(_to, _amount);
+
+        emit CrosschainMint(_to, _amount);
+    }
+
+    /// @notice Allows the TOKEN_BRIDGE to burn tokens.
+    /// @param _from   Address to burn tokens from.
+    /// @param _amount Amount of tokens to burn.
+    function crosschainBurn(address _from, uint256 _amount) external onlyTokenBridge {
+        _burn(_from, _amount);
+
+        emit CrosschainBurn(_from, _amount);
```

Comment:
```suggestion
        emit CrosschainBurn(_from, _amount);
```

This is an error if the plan is to keep `CrosschainBurn` in this [comment](https://github.com/ethereum/ERCs/pull/692/files#r1823583714).

Reply:
correct is `CrosschainBurn`. Will resolve